src/test/resources/testData/sipp/dissemination.xml

```diff
@@ -3,48 +3,48 @@
   <Abstract>There is an increasing need to provide actionable science for water management planning and ecological drought preparedness. However, questions remain regarding the most effective and efficient methods for extending scientific knowledge and products into management action and decision-making. This study analyzed two unique cases of water management in the context of ecologically available water to understand the translation of scientific knowledge into management. In particular, the research examined and compared (1) characteristics of the science being assessed and applied and (2) ideal types of scientific knowledge or products that facilitate the translation process towards action, management, and decision-making. The first case, beaver mimicry, is an emerging nature-based solution used to restore riparian areas, increase groundwater infiltration, and slow surface water flow that is rapidly being adopted by the natural resource management community. The second, Colorado Dust on Snow, is an established research program funded by several agencies and water conservation districts that provides water managers with scientific information regarding how movement of dust influences hydrology and timing of water runoff in critical water sources. For each case, ethnographic conversations with scientists and practitioners were used to understand how scientific knowledge translates into action and decision making. Conversations were transcribed and analyzed using qualitative methods. Results explore how the salience, credibility, and legitimacy of information is viewed differently by scientists and practitioners. A manuscript describing these results is in preparation for the journal Environmental Management.</Abstract>
   <BasisNumber>RB00H5X.2</BasisNumber>
   <CostCenter>Fort Collins Science Center</CostCenter>
-  <IPPANumber xsi:nil="true" />
```

Comment:
This is the actual input for the test from the real SIPP process. Please revert to the original data.